src/engine/Server.cpp

```diff
@@ -529,7 +542,7 @@ CPP_template_2(typename RequestT, typename ResponseT)(
       std::move(parsedHttpRequest.operation_),
       ad_utility::OverloadCallOperator{visitQuery, visitUpdate, visitGraphStore,
                                        visitNone},
-      requestTimer, request, send);
+      requestTimer, request, send, plannedQuery);
```

Comment:
might take a `PlannedQuery` instead of `optional` (check and review)

Reply:
We cannot get rid of the `optional`. A value will only be set once a `visit...` is actually run inside `processOperation` and `visitNone` does not provide a `PlannedQuery`.